task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -96,6 +96,21 @@ spec:
               exit 0
             fi
 
+            timeout=1200  # 20 minutes 
+            interval=60  # interval between checks in seconds
```

Comment:
Maybe we can lower this interval so we don't end up waiting for up to an extra minute for the database to be ready. Something along 15-20 seconds might work.

Reply:
Sure thing!

Reply:
I've kept the interval as 20 seconds and now only output the message to the log every 60 seconds. What do you think?